plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/procedure/IcebergOptimizeHandle.java

```diff
@@ -97,4 +97,15 @@ public boolean isRetriesEnabled()
     {
         return retriesEnabled;
     }
+
+    @Override
+    public String toString()
+    {
+        return new StringBuilder()
+                .append("procedure").append(":").append("optimize")
```

Comment:
generate the toString using Intellij,  prefer Guava's `toStringHelper`  (except in SPI where Guava is not on the pass, use StringJoiner there)

```
    @Override
    public String toString()
    {
        return toStringHelper(this)
                .add("schemaAsJson", schemaAsJson)
                .add("partitionSpecAsJson", partitionSpecAsJson)
                .add("tableColumns", tableColumns)
                .add("fileFormat", fileFormat)
                .add("tableStorageProperties", tableStorageProperties)
                .add("maxScannedFileSize", maxScannedFileSize)
                .add("retriesEnabled", retriesEnabled)
                .toString();
    }
```